File: src/main/java/com/procurement/clarification/ClarificationApplication.java
package com.procurement.clarification;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;

@SpringBootApplication
@EnableEurekaClient
public class ClarificationApplication {
    public static void main(final String[] args) {
        SpringApplication.run(ClarificationApplication.class, args);
    }
}

File: src/main/kotlin/com/procurement/clarification/ClarificationApplication.kt
package com.procurement.clarification

import com.procurement.clarification.infrastructure.configuration.ApplicationConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackageClasses = [ApplicationConfig::class])
class ClarificationApplication

fun main(args: Array<String>) {
    runApplication<ClarificationApplication>(*args)
}